crates/wascc-provider/src/lib.rs

```diff
@@ -108,7 +108,7 @@ impl Provider for WasccProvider {
             .get(ACTOR_PUBLIC_KEY)
             .map(|a| a.to_string())
             .unwrap_or_else(|| "".into());
-
+        debug!("{:?}", pubkey);
```

Comment:
Did you want this to be echoed during debug?

Reply:
removed it, but it is really nice seeing the public key of the actor as you go through debugging